packages/wrangler/src/__tests__/pages/deploy.test.ts

```diff
@@ -5510,7 +5510,9 @@ Failed to publish your Function. Got error: Uncaught TypeError: a is not a funct
 				expect(contents).toContain(
 					'Content-Disposition: form-data; name="functionsWorker-0.test.js.map"'
 				);
-				expect(contents).toContain('"sources":["[[path]].ts"');
+				expect(contents).toContain(
+					`"sources":["${encodeURIComponent("[[path]].ts")}"`
```

Comment:
[esbuild treat source map path references as URLs from v0.25.0](https://github.com/evanw/esbuild/releases/tag/v0.25.0)

Reply:
I wonder if this will fix the warnings I keep seeing in the Miniflare tests that complain that source maps cannot be found...?